aws/files/autoeips.py

```diff
@@ -74,10 +120,21 @@ def associate_eip(self, eips, retries=3):
 
                 success = eip.associate(
                     instance_id=self.instance_id, allow_reassociation=False)
+                self.update_standby_mode(False)
                 return success
+        logger.warning("Failed to associate with any EIP's")
+        self.update_standby_mode(True)
         return False
 
     def get_unassociated_eips(self):
+        """
```

Comment:
yep, just noticed that tiredness failure :) If I can ignore the pythonic though please. I find len(self.filter_addresses) < 1 just *so* much clearer.